pkg/providers/aws/cluster_network_provider.go

```diff
@@ -56,13 +83,135 @@ func (n *NetworkProvider) CreateNetwork() error {
 		- Check if a VPC exists with the integreatly.org/clusterID tag
 		- If VPC doesn't exist, create a VPC with CIDR block and tag it
 	*/
-	logger := n.logger.WithField("action", "CreateNetwork")
+	logger := resources.NewActionLogger(n.logger, "CreateNetwork")
 	logger.Debug("CreateNetwork stub")
-	return errorUtil.New("CreateNetwork stub")
+	//return nil, errorUtil.New("CreateNetwork stub")
+	// for development, replace 'placeholder with a real vpc id until vpc creation is merged
+	return &Network{Vpc: &ec2.Vpc{VpcId: aws.String("placeholder")}}, nil
```

Comment:
Will wait on VPC creation to be merged before merging this, so this will be gone